src/main/java/core/basesyntax/service/ShopService.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import java.util.Map;
+
+public interface ShopService {
+    void process(List<FruitTransaction> transactions);
+
+    Map<String, Integer> getStorage();

```

Comment:
It's better to encapsulate the storage details and not expose them via the interface. Consider providing a method that can report the contents of the storage without giving direct access to the underlying data structure.